src/android/CodePush.java

```diff
@@ -518,7 +522,18 @@ private void navigateToFile(File startPageFile) throws MalformedURLException {
     private void navigateToURL(String url) {
         if (url != null) {
             CodePush.ShouldClearHistoryOnLoad = true;
-            this.mainWebView.loadUrlIntoView(url, false);
+            if (this.hasIonicWebViewEngine()) {
+                try {
+                    String ionicWebViewEngineUrlPath = new URI(url).getPath();
+                    String ionicWebViewEngineServerPath = ionicWebViewEngineUrlPath.substring(0, ionicWebViewEngineUrlPath.indexOf("/index.html"));
+                    this.setServerBasePath(ionicWebViewEngineServerPath);
+                    return;
```

Comment:
@atftech-artyom @whegar It looks like unnecessary `return;`. Please let me know if I'm wrong and miss something or just remove it.